### ISSUE TITLE
the label_smoothing argument for CE loss in keras

### ISSUE THREAD

**Issue by yjiangling** [tensorflow/tensorflow#46849]
Recently, I find that only the loss function 

tf.keras.losses.categorical_crossentropy(y_true,y_pred,from_logits=True,label_smoothing=0.1)

has argument "label_smoothing", 

both function:  tf.keras.losses.sparse_categorical_crossentropy
and class:  tf.keras.losses.SparseCategoricalCrossentropy

do not have this parameter.

Why not add "label_smoothing" to this CE loss functions?

**Comment by vaibhavarora102** [comment]
Hi, @Saduf2019 
I had been using TensorFlow for machine learning for a little while and feel confident upon the good hold over TensorFlow.
I would like to resolve this issue with your guidance. 
If you feel that It could be the best issue for a user like me who is going to contribute to TensorFlow for the first time kindly assign me this issue.
thanks

**Comment by krenerd** [comment]
Isn't label_smoothing related with the data itself, not the loss? 
Comparing `ce(smoothed labels, pred)` instead of `(0/1 labels, pred)`. This is related with the label data you feed in, instead of the loss function.
